src/app/container/container.component.ts

```diff
@@ -146,10 +146,27 @@ export class ContainerComponent extends Entry implements AfterViewInit {
     // messy prototype for a carousel https://developers.google.com/search/docs/guides/mark-up-listings
     // will need to be aggregated with a summary page
     this.schema = {
-      '@type': 'ListItem',
-      'position': this.tool.id,
-      'url': this.shareURL
+      '@type': 'SoftwareApplication',
+      'description': this.tool.description,
+      'name': this.tool.name,
+      'softwareVersion': this.defaultVersion,
+      'url': window.location,
```

Comment:
I'm not sure if this shouldn't be the TRS url, although it's not clear to me if it should be the UI or the backend URL.

Also, this won't be consistent if we decided to go with the UI url; if I bookmark a tab, e.g., [this](https://dockstore.org/containers/quay.io/pancancer/pcawg-dkfz-workflow:2.2.0?tab=launch), it will have a different url than if I go to the info page.